db_stress_tool/db_stress_gflags.cc

```diff
@@ -28,6 +28,18 @@ DEFINE_bool(read_only, false, "True if open DB in read-only mode during tests");
 DEFINE_int64(max_key, 1 * KB * KB,
              "Max number of key/values to place in database");
 
+DEFINE_int32(max_key_len, 3,
```

Comment:
I don't see the point of this option if it has to agree with the number of values in key_len_percent_dist.

It also seems easy and valuable to use key lengths that are not multiples of 8, such as appending 0-7 'x's depending on the bottom bits of offset.

Reply:
I meant to make the key_len_percent_dist optional, but just realized I haven't taken care of initializing it if not specified.

The idea of key lengths that are not multiples of 8 byb appending 0-7 'x's is good. I'll implement it.